feature/feature.go

```diff
@@ -47,6 +47,11 @@ const (
 	// owner: @upxinxin
 	// alpha: v1.8
 	EdgeZone featuregate.Feature = "EdgeZone"
+
+	// NodePrototyping is the feature gate for having prototyping mode on
+	// owner: @bennycortese
+	// alpha: v0.1
```

Comment:
This should be the CAPZ version where this was first introduced, which we expect to be the next minor release:
```suggestion
	// alpha: v1.11
```